### ISSUE TITLE
1.8beta1 delete+download on move

### ISSUE THREAD

**Issue by labkode** [owncloud/client#2853]
I tried the following test against the demo server at demo.owncloud.org (8.0.0) and a custom one (7.0.1).
When you move a folder inside another in the web interface the client does not follow the move and sees a DELETE + DOWNLOAD of the contents of the folder.

With version 1.7.1 this not happen.
Maybe this issue is related with #2852

**Comment by moscicki** [comment]
BTW, this is a blocker for us. Only last week we have seen 3K directory trees moved/renamed (and possibly countless files contained within these directories).

**Comment by ogoffart** [comment]
I cannot reproduce the problem.
This should have been working for a few versions now

**Comment by dragotin** [comment]
@labkode can you produce a log that isolates the problem?

**Comment by labkode** [comment]
Klaas I tested this morning in the MAC osx client as well against demo.owncloud.org, can you tell me which log are you interested in ? The one with the flag --log-file ?

**Comment by dragotin** [comment]
I tested with demo.owncloud.com as well. I moved the *Music* folder into *Photos* and the client indeed started to download these, however, the Music folder reappeared etc. I got strange error messages ("Request Entity Too Large") for the re-download of the original Music files etc. pp.

BUT - demo.o.c is not a normal ownCloud server, I think it resets its database and files every couple of minutes, so that might confuse the client.

The question is if you also could see this behaviour with a "normal" owncloud server, @labkode ?
